src/base/system_parameter.c

```diff
@@ -2340,6 +2342,10 @@ static int prm_regexp_engine_upper = cubregex::engine_type::LIB_RE2;
 static unsigned int prm_regexp_engine_flag = 0;
 /* *INDENT-ON* */
 
+bool PRM_ORACLE_STYLE_NUMBER_RETURN = false;
+static bool prm_oracle_style_number_return_default = false;
```

Comment:
I recommend to change the default to true and perform a SQL test at once.
By doing that, I think unexpected problems may be discovered.
I worry that the changes are not tested at all in SQL test.

Reply:
Yes, I have already tested and I got the expected results which are just modification by printing out without trailing zeros.